paws-of-luv-frontend/src/App.js

```diff
@@ -1,26 +1,75 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { Component } from 'react';
 import './App.css';
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch
+} from 'react-router-dom'
+import mockCats from './mockCats.js'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import Header from './components/Header'
+import Footer from './components/Footer'
+
+import Home from './pages/Home'
+import CatIndex from './pages/CatIndex'
+import CatShow from './pages/CatShow'
+import CatNew from './pages/CatNew'
+import CatEdit from './pages/CatEdit'
+import NotFound from './pages/NotFound'
+
+
+class App extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      cats: mockCats
+    }
+  }
+
+  componentDidMount() {
```

Comment:
I would highly recommend moving this styling into the css file. Referencing the `document.body` is called directly manipulating the DOM which can cause unexpected behavior.